src_base/xeve_picman.c

```diff
@@ -243,7 +243,7 @@ void xeve_copy_refp(XEVE_REFP * refp_dst, XEVE_REFP * refp_src)
     refp_dst->list_poc = refp_src->list_poc;
 }
 
-int check_copy_refp(XEVE_REFP(*refp)[REFP_NUM], int cnt, int lidx, XEVE_REFP  * refp_src)
```

Comment:
We may not need this fuction, how about remove this?

Reply:
Removing this function is beyond the scope of this PR.
However, this is not a public API function, and moreover I don't see any use cases for this function in the library either. 
It looks like dead code so the function could be removed in the next PR.